_posts/2016-03-30-3d-elevation-program-proposal.md

```diff
@@ -11,6 +11,6 @@ categories: []
 tags: []
 ---
 
-[![3D Elevation Program]({{ "/images/3DEP_Proposal_sm.png" | prepend: site.baseurl }})]({{ "/images/3DEP_Proposal.png" | prepend: site.baseurl }})
+[![3D Elevation Program]({{ "/images/3DEP_Proposal_sm.png" | prepend: site.baseurl }}'click to enlarge')]({{ "/images/3DEP_Proposal.png" | prepend: site.baseurl }})
```

Comment:
you want to have a space between the `}}` and the `'title text'`. I fixed it for you.